projects/laji/src/app/shared-modules/own-submissions/service/document.service.ts

```diff
@@ -51,13 +51,13 @@ export class DocumentService {
   }
 
   deleteDocument(id: string) {
-    return this.documentApi.delete(id, this.userService.getToken()).pipe(
-      mergeMap(() => this.userService.user$),
-      tap(person => {
+    const isTmpDoc = id.match(/^T:\d+$/);
+    const del$ = isTmpDoc
+      ? this.userService.user$.pipe(tap(person => {
```

Comment:
I think this should have `user$.pipe(take(1))` because we only want the current user

Reply:
We do that in numerous places, but I'm pretty sure it's unnecessary. Logging in/out reloads the page and thus the `user$` pipe will always return just a single user?

Reply:
OK I tested that logging out doesn't reload the page so better add the `take(1)` so that logging out won't do anything weird